manim/scene/scene_file_writer.py

```diff
@@ -494,6 +494,16 @@ def combine_movie_files(self):
             '-c', 'copy',
             movie_file_path
         ]
+        if not self.save_as_gif:
+            commands+=[
+                "-c", "copy",
+                movie_file_path
+            ]
+        if self.save_as_gif:
+            movie_file_path=self.gif_file_path
+            commands += [
```

Comment:
so saving as GIF will not save as mp4, right? We should document this somewhere...

Reply:
That _is_ the intended behaviour, and the latest commit _should_ make that happen.
However, since the default behaviour for manim is write_to_movie, it also makes the MP4 file and shows that as well.